docs/tutorial.md

```diff
@@ -614,6 +614,7 @@ information. The file should have the following columns, separated by `;`:
 | `type` | type of this feature; see below |
 | `function` | function description of this feature or functional annotation; see below |
 | `reference` | reference to the literature for this feature (if any) |
+| `genomic_mutations` | metadata for mutations; see below |
```

Comment:
does the column need to be present even if we don't e.g. annotate mutations, or we don't use metadata? We should verify this and specify it in the docs. I think that if possible we shouldn't check for it if we're not using the metadata

Reply:
As it is now, the file is parsed by itself, and the metadata is only specified when adding mutations. To not parse the genomic_mutations column the user would need to specify the metadata when parsing the file, I think. Can you think of another way?

Reply:
Maybe we can parse the file without the column, and if the genomic_mutations is specified in metadata, then we can parse the column in add_mutations